File: src/test/java/no/nav/sosialhjelp/soknad/web/utils/FeatureToggleUtilsTest.java
package no.nav.sosialhjelp.soknad.web.utils;

import org.junit.jupiter.api.Test;

import static org.assertj.core.api.Assertions.assertThat;

class FeatureToggleUtilsTest {

    @Test
    void enableModalV2Test() {
        assertThat(FeatureToggleUtils.enableModalV2("1234")).isTrue();
        assertThat(FeatureToggleUtils.enableModalV2("1235")).isFalse();
        assertThat(FeatureToggleUtils.enableModalV2("asdf")).isFalse();
        assertThat(FeatureToggleUtils.enableModalV2(null)).isFalse();
        assertThat(FeatureToggleUtils.enableModalV2("01234")).isTrue();
    }
}

File: src/test/kotlin/no/nav/sosialhjelp/soknad/api/featuretoggle/FeatureToggleUtilsTest.kt
package no.nav.sosialhjelp.soknad.api.featuretoggle

import io.mockk.mockk
import no.finn.unleash.Unleash
import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

internal class FeatureToggleUtilsTest {

    private val unleash: Unleash = mockk()
    private val featureToggleRessurs = FeatureToggleRessurs(unleash)

    @Test
    fun leggTilBarnTest() {
        val response = featureToggleRessurs.featureToggles()
        assertThat(response["leggeTilBarn"]).isFalse
    }
}
